src/ddmd/target.d

```diff
@@ -284,6 +285,82 @@ struct Target
         return 0;
     }
 
+    /**
+     * Checks whether the target supports operation `op` for vectors of type `type`.
+     * For binary ops `t2` is the type of the 2nd operand.
+     *
+     * Returns:
+     *      true if the operation is not supported.
```

Comment:
`true` if `not` is awkward, as well as the `!supported` on line 361. How about changing the name to `isVectorOpSupported` then the use of true, false, and nots become straightforward. Or maybe `hasVectorSupport` which is a bit shorter. I know I've written a lot of `check` functions, but perhaps they are misnamed as well.